src/bsp/src/stm32/src/interloc/CMakeLists.txt

```diff
@@ -44,6 +48,7 @@ target_include_directories(${LIB_NAME}
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
         $<INSTALL_INTERFACE:include>
         ${BSP_CMAKE_DIR}/include
+        ${BSP_PLATFORM_CMAKE_DIR}/include
```

Comment:
Why are we including a dir instead of using a library?

Reply:
Because I need the private headers of the lib. Technically this cmake is still part of the BSP::Platform lib. It's only in a separate folder because there are a lot of files.